### PR TITLE
Switch to new wof parent schema

### PR DESCRIPTION
**Pull request by dianashk**
- [x] Rename alpha3, admin0, admin1 to country_a, country, region.
- [x] Update label generation to use new properties
- [x] Update query building to use new pelias-query property names
- [x] Update pelias-query version

Connected to #424

### CODE REVIEW

**Comment by orangejulius** [comment]
I was able to rebase this onto my cleaned up branch with `git rebase --onto new_wof_model_format 3e08138` and the changes look good so far

**Comment by dianashk** [comment]
Awesome thanks!

File: middleware/renamePlacenames.js [hunk 1]
```diff
@@ -36,31 +51,27 @@ function renamePlacenames(req, res, next) {
...
-    if (property === 'address') {
-      extend(newPlace, renameProperties(place[property]));
-    }
-    else {
-      renameProperty(place, newPlace, property);
-    }
-  });
-  return newPlace;
-}
+function renameStuff(place) {
```
Comment (dianashk) [inline middleware/renamePlacenames.js]: add comment and maybe rename "renameStuff"

File: middleware/renamePlacenames.js [hunk 2]
```diff
@@ -11,19 +11,34 @@ var extend = require('extend');
  - A is an abbreviation or code for the place (e.g. "NYC" for New
  York)
  */
-// config mapping of old names to new ones
-var NAME_MAP = {
+
+var ADDRESS_PORPS = {
```
Comment (orangejulius) [inline middleware/renamePlacenames.js]: PORPS!

Comment (dianashk) [inline middleware/renamePlacenames.js]: misspelled PROPS

File: package.json [hunk 3]
```diff
@@ -53,7 +53,7 @@
     "pelias-config": "^1.0.1",
     "pelias-esclient": "0.0.25",
     "pelias-logger": "^0.0.8",
-    "pelias-query": "^5.1.0",
+    "pelias-query": "^6.0.0",
```
Comment (dianashk) [inline package.json]: make version explicit

File: public/attribution.md [hunk 4]
```diff
@@ -3,5 +3,5 @@
 * Data from
    * [OpenStreetMap](http://www.openstreetmap.org/copyright) © OpenStreetMap contributors under [ODbL](http://opendatacommons.org/licenses/odbl/)
    * [OpenAddresses](http://openaddresses.io) under a [Creative Commons Zero](https://github.com/openaddresses/openaddresses/blob/master/sources/LICENSE) public domain designation
-   * [Quattroshapes](https://github.com/foursquare/quattroshapes/blob/master/LICENSE.md) under [CC-BY-2.0](https://creativecommons.org/licenses/by/2.0/)
    * [GeoNames](http://www.geonames.org/) under [CC-BY-3.0](https://creativecommons.org/licenses/by/2.0/)
+   * [WhosOnFirst](http://whosonfirst.mapzen.com)
```
Comment (dianashk) [inline public/attribution.md]: add WOF license

File: package.json [hunk 5]
```diff
@@ -53,7 +53,7 @@
...
     "pelias-schema": "1.0.0",
```
Comment (orangejulius) [inline package.json]: If pelias-schema is no longer used, can we remove it from dependencies?

File: test/unit/actual.js [hunk 6]
```diff
@@ -0,0 +1,65 @@
+var actual = {
```
Comment (dianashk) [inline test/unit/actual.js]: remove this file

File: test/unit/expected.js [hunk 7]
```diff
@@ -0,0 +1,65 @@
+var expected = {
+  bbox: [-73.985656, 40.748432, -0.101795, 51.5337144],
```
Comment (dianashk) [inline test/unit/expected.js]: remove this file

**Pull request merged**